src/layouts/layouts.css

```diff
@@ -75,8 +75,8 @@
 }
 
 .layout-2-1 > div {
-    flex: 1;
-    max-height: 50%;
+    flex: 1 1 50%;
+    min-height: 0;
```

Comment:
@borisyankov  yup. Apparently the flex-basis doesn't work without it. It took me long time to realised that. You can try 😉